lib/puppet/type/gluster_volume.rb

```diff
@@ -80,6 +80,19 @@ def property_matches?(current, desired)
     munge { |val| Array(val) }
   end
 
+  newparam(:local_peer_aliases, :array_matching => :all) do
+    desc([
+        'Other names for the current host.',
+        'Anything included here will be ignored when checking peers.',
+      ].join(' '))
```

Comment:
I'd prefer this to be a single line and I don't really like any of the available multiline syntaxes. I suppose I could use `%q()` or something, though.